### PR TITLE
Fixes and clean

### PR DESCRIPTION
**Pull request by psolyca**
Some cleaning and fixes on the code before going futher.

### CODE REVIEW

File: sample_data/leica_1205_1.txt [hunk 1]
```diff
@@ -1,2365 +0,0 @@
-                              System 1200 Data Export - File Begin
```
Comment (steko) [inline sample_data/leica_1205_1.txt:1]: Why delete this file?

File: sample_data/zeiss_elta_r55/zeiss_elta_r55-REC_500.tops [hunk 2]
```diff
@@ -1,129 +1,129 @@
-   0001 OR.COOR
```
Comment (steko) [inline sample_data/zeiss_elta_r55/zeiss_elta_r55-REC_500.tops]: What's going on with these changes? Is this caused by newlines? I think it's better to leave these untouched.

File: totalopenstation/formats/__init__.py [hunk 3]
```diff
@@ -56,30 +56,27 @@ class FeatureCollection(g.FeatureCollection):
     pass
 
 
-class Parser:
+class Parser(object):
```
Comment (steko) [inline totalopenstation/formats/__init__.py]: I don't think this is necessary since all classes are new style classes in Python 3

Comment (psolyca) [reply totalopenstation/formats/__init__.py]: Yes, these changes were made before py3 migration ^^
I drop some py2-py3 changes on these commits but not all.
Will revert those.

File: totalopenstation/formats/leica_gsi.py [hunk 4]
```diff
@@ -325,11 +327,11 @@ def raw_line(self):
         '''
 
         points = []
-        ldata = len(self.line[0].split()[0].lstrip('*')[7:])
+        ldata = len(self.rows[0].split()[0].lstrip('*')[7:])
```
Comment (steko) [inline totalopenstation/formats/leica_gsi.py:330]: I don't remember what this line does, perhaps we can add a comment to explain or give the variable a more explicit name?

File: sample_data/zeiss_elta_r55/zeiss_elta_r55-REC_500.tops (hunk 2 above)
Comment (psolyca) [reply sample_data/zeiss_elta_r55/zeiss_elta_r55-REC_500.tops]: Nothing, I did not touch this file, I just move it.
I think it is the automatic change of git for line ending !

File: totalopenstation/formats/trimble_are.py [hunk 5]
```diff
@@ -25,19 +25,16 @@
...
-        Parser.__init__(self, data)
-
     def is_point(self, line):
         is_point = False
         if "5=" and "4=" and "37=" and "38=" and "39=" in line:
             is_point = True
         return is_point
 
-    def get_point(self, chunk):
+    def get_point(self, line):
```
Comment (steko) [inline totalopenstation/formats/trimble_are.py]: If you look at the sample data, e.g. https://github.com/steko/totalopenstation/blob/master/sample_data/trimble/BSG-08-11-19.are there is one "value" per line and that to obtain a point you have to parse several lines. The `line` variable name seems misleading.

**Review by steko — CHANGES_REQUESTED** [review]
Everything looks good, I put some minor comments here and there and there are a few sample data files that have unnecessary changes, probably for newlines.

Thank you!

File: sample_data/leica_1205_1.txt (hunk 1 above)
Comment (psolyca) [reply sample_data/leica_1205_1.txt:1]: Cause it is the same file as this one https://github.com/steko/totalopenstation/blob/master/sample_data/leica_tcr_1205

File: totalopenstation/formats/leica_gsi.py (hunk 4 above)
Comment (psolyca) [reply totalopenstation/formats/leica_gsi.py:330]: This variable is used to get the block code length (8 or 16).
It is not used anymore in points method but it is in the raw_line one.
I will drop it in points method.

File: totalopenstation/formats/trimble_are.py (hunk 5 above)
Comment (psolyca) [reply totalopenstation/formats/trimble_are.py]: Ok, I will revert it.

File: sample_data/zeiss_elta_r55/zeiss_elta_r55-REC_500.tops (hunk 2 above)
Comment (psolyca) [reply sample_data/zeiss_elta_r55/zeiss_elta_r55-REC_500.tops]: I tried all my settings in Git for line endings and... those 2 files are always converted !

Comment (psolyca) [reply sample_data/zeiss_elta_r55/zeiss_elta_r55-REC_500.tops]: Done !

**Comment by psolyca** [comment]
Now everything should be ok.

**Review by steko — APPROVED** [review]
Good to go!

**Pull request merged**